libr/asm/p/asm_arm_cs.c

```diff
@@ -170,9 +170,10 @@ static int hack_handle_ldst(ut32 insn, char **buf_asm) {
 				mnemonic = "ldgm";
 				break;
 			}
-
-			if (!imm9) {
-				*buf_asm = sdb_fmt ("%s x%d, [x%d]", mnemonic, Xt, Xn);
+			if (!strcmp (mnemonic, "stgm") || !strcmp (mnemonic, "stzgm") ||
```

Comment:
`!strcmp` can be replaced with `op2 ==` but would impact readability IMO. what do you think?

Reply:
i think you can set a bool var before the switch and just chk for it. also, as long as you are settings those strings inline you can compare pointers instead. but a bool var would be cleaner imho